example/app/data/DemoDataSource/plugins/form/simple/simple.json

```diff
@@ -2,5 +2,5 @@
   "_id": "Simple",
   "name": "Simple",
   "type": "./blueprints/Simple",
-  "Firstname": ""
+  "stringOptional": "a prefilled optional string"
```

Comment:
This does not seem to be a correct implementation of Simple. It lacks the non-optional attributes `stringRequired` and `requiredCheckbox`, and it contains `name`. which is not defined in the blueprint.